FF1Lib/LegendaryShops.cs

```diff
@@ -10,22 +10,19 @@ namespace FF1Lib
 {
 	public class LegendaryShops
 	{
-		MT19337 rng;
-		Flags flags;
-		FF1Rom rom;
-		List<Map> maps;
-
-		MapTileSets MapTileSets;
-		ShopData ShopData;
-		TileSet[] TileSets = new TileSet[8];
-
-		Dictionary<byte, List<byte>> UnusedTilesbyTileSet;
-		Dictionary<string, MagicSpell> Spells;
-
-		List<SpellInfo> SpellInfos;
+		private readonly MT19337 rng;

```

Comment:
if you kill my whitespace, im leaving...

Just joking, but in my opinion this goes too far.

Reply:
The rules are up for tweaking, I'll try to find the rule that enforces no blank lines here and we can discuss it as a group.

Reply:
thanks

Reply:
Oh, yeah.  Single blank lines should always be acceptable.  I'm okay with the linter removing multiple blank lines or blank lines before close braces.